kin-devplatform-sdk/src/main/java/kin/devplatform/data/blockchain/KeyStoreProviderImpl.java

```diff
@@ -51,9 +51,11 @@ public int importAccount(@NonNull final String keystore, @NonNull final String p
 		Validator.checkNotNull(keystore, "password");
 		try {
 			IKinAccount importedAccount = kinClient.importAccount(keystore, password);
+			String importedPublicAddress = importedAccount.getPublicAddress();
 			int index = -1;
 			for (int i = 0; i < kinClient.getAccountCount(); i++) {
-				if (importedAccount == kinClient.getAccount(i)) {
+				if (importedPublicAddress != null &&
```

Comment:
So it was a problem of '==' equality instead of equals because of different objects after the migration module was integrated...